src/main/filesystem.ts

```diff
@@ -70,6 +72,20 @@ export class KangarooFileSystem {
   keystoreInitialized = () => {
     return fs.existsSync(path.join(this.keystoreDir, 'lair-keystore-config.yaml'));
   };
+
+  readOrCreatePassword() {
+    const pwPath = path.join(this.appDataDir, '.pw');
+    if (!fs.existsSync(pwPath)) {
+      const pw = uuidv4();
+      fs.writeFileSync(pwPath, pw, 'utf-8');
```

Comment:
I see you are storing the password in plain text, guessing this is okay for now, though not the most way to go about it

Reply:
There is no other way actually without a user-provided password. Otherwise it would need to be encrypted which would again require that a key for encryption is stored or hard-coded somewhere...I think there is no golden bullet here and this was considered better than a hard-coded password when we discussed it during refinement...

Reply:
Yea, I can see the complexity growing if we choose to go this route, I think it would be up to the happ dev on the best way to go about this, maybe adding a note about this in the README would be helpful